policy-author/README.md

```diff
@@ -0,0 +1,125 @@
+# Policy-Author API (draft)
+
+## Background
+
+The Policy API specifies an endpoint and a JSON schema for communicating mobility policy to providers. This document describes a companion API to the provider-facing Policy endpoints for creating and editing Policy and Geography objects.
+
+[Policy PR](https://github.com/CityOfLosAngeles/mobility-data-specification/pull/322)
+
+## Endpoints
+
+The agency-facing Policy Author API consists of the following endpoints. The endpoints to list Policy and Geography objects are defined in the Policy API. The only addition is that when calling the GET /policies endpoint, append ?unpublished to get the unpublished ones, separate from the published ones. Adding ?unpublished will have no effect if your access token has a provider_id.
+
+### Policy Endpoints
+
+Responses must set the `Content-Type` header, as specified in the [Provider versioning](../provider/README.md#versioning) section. They must also specify the API version in the JSON-formatted response body, under the `version` key.
+
+#### Schema
+
+See Policy API.
+
+Metadata is free-form JSON format.
+
+### GET /policies
+
+Get a list of policies.
+
+A note: the `get_published` and `get_unpublished` parameters only make sense in the Policy Author version of this endpoint, because the Policy Author API is intended for use by Agencies and not Providers. Providers should never be able to see unpublished Policies, as those are not yet meant for public consumption.
+
+Parameters:
+| Name         | Type      | R/O | Description                                    |
+| ------------ | --------- | --- | ---------------------------------------------- |
+| `id`         | UUID      | Optional    | If provided, returns one policy object with the matching UUID; default is to return all policy objects.                       |
```

Comment:
This seems wrong, it should definitely be a `policy_id` parameter (which is what it is in the reference impl)